BackofficeBundle/Security/Authorization/Voter/NodeGroupRoleVoter.php

```diff
@@ -90,4 +101,28 @@ public function vote(TokenInterface $token, $object, array $attributes)
 
         return self::ACCESS_ABSTAIN;
     }
+
+    /**
+     * @param NodeInterface  $node
+     * @param GroupInterface $group
+     * @param string         $attribute
+     *
+     * @return boolean
+     */
+    protected function isGrantedNodeGroupRole(NodeInterface $node, GroupInterface $group, $attribute)
+    {
+        $nodeGroupRole = $group->getNodeRoleByNodeAndRole($node->getNodeId(), $attribute);
+        if ($nodeGroupRole instanceof NodeGroupRoleInterface) {
+            if (NodeGroupRoleInterface::ACCESS_INHERIT === $nodeGroupRole->getAccessType()) {
+                $nodeParent = $this->nodeRepository->findInLastVersion($node->getParentId(), $node->getLanguage(), $node->getSiteId());
+                if (null !== $nodeParent) {
+                    return $this->isGrantedNodeGroupRole($nodeParent, $group, $attribute);
```

Comment:
@itkg-ngilain  it's not obligatory to add an NL before the return if there is one line in if